Assets/MRTK/Tools/package.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "com.microsoft.mixedreality.toolkit.tools",
+    "displayName": "Mixed Reality Toolkit Tools",
+    "version": "2.5.0-preview.0",
+    "unity": "2019.4",

```

The others (other than XR Plugin) are 2018.4. Should this one be too?

```suggestion
    "unity": "2018.4",

```

---

great catch. thanks!